wrangler-proto/src/main/java/io/cdap/wrangler/proto/workspace/v2/Workspace.java

```diff
@@ -164,4 +177,15 @@ public Workspace build() {
                            insights);
     }
   }
+
+  /**
+   * UserDefinedAction enum.
+   */
+  public enum UserDefinedAction {
+    NO_ACTION,
+    SKIP_ROW,
+    SEND_TO_ERROR_COLLECTOR,
+    ERROR_PIPELINE,
+    NULLABLE
```

Comment:
How would Nullable be different from No_Action ?

Reply:
It is not different, both these options will return the row as it is. I have kept a NULLABLE option for columns that were non- nullable earlier and were converted to Nullable column later. I might remove this option later if not required.